EDI/Operation/04_ManageLineItemsFileStt.md

```diff
@@ -0,0 +1,185 @@
+    - The *EDI export SG26/SG27* entry is available.
+
+        ![SG26 Mapping](../../Assets/Screenshots/EDI/Operation/ExportDefMapSG26Segment.png "[SG26 Mapping]")
+
+6. Click the *EDI Export SG26/SG27* destination attribute and define the mapping.    
+   The settings section is displayed on the right.
+
+    ![SG26 Mapping settings](../../Assets/Screenshots/EDI/Operation/ExportDefSG26MappingKonfig.png "[SG26 mapping settings]")
+   
+     -  Click the *Extension* drop-down list and select the **XML SubExport extension**.   
+     Both the "Source attribute" and the *Configuration* section is displayed in the left part of the *Settings* section.
+
+     - Click the *Override change tracking mode* drop-down list and select the desired one.
+
+     -  On the left part of the *Settings* section, click the *Source attribute* drop-down list and select **Positions** or another attribute that represents a line item. 
+
+     -  In the *Configuration* section, click the *Subexport definition* drop-down list and select the required line item definition, for example **EDIFACT INVOIC export items customer 12.**
+     
+        ![SG26 Mapping settings](../../Assets/Screenshots/EDI/Operation/ExportDefSG26MappingDone.png "[SG26 mapping settings]")  
+
+7. Click the [SAVE] button in the top right corner of the wizard.   
+    The *Please select a destination attribute in the grid on the left to edit the mapping* text is displayed.
+
+8. Finalize the *Map attributes* wizard by clicking the [CONTINUE] button until you can click the [FINALIZE] button.
+  
+9. Click the [SAVE] button in the top right corner to save the header definition.   
+   The *Exporter definitions* view is displayed.
+
+
+## Get line item segments
+
+Get the line item segments of an EDIFACT message in order to be able to fill them. You can add constant values as well as strings and conditions that are to be determined via attribute mapping later. 
+
+
+#### Prerequisites
+
+- You have the message specification of your business partner at hand.
+- You are currently creating the line item definition and have entered the necessary data in the *Settings* tab, see [Create basic definition settings](./01_ManageDefinitions.md#create-basic-definition-settings).
+- You have prepared the line item export in the header file definition, see [Prepare line item export](./02_ManageHeaderFileSett.md#prepare-line-item-export).
+- You have prepared the data mapping for the line items, [Prepare data mapping for the line items](#prepare-data-mapping-for-the-line-items).
+
+#### Procedure
+
+*DataHub Exporter > EXPORTS tab > Settings tab*
+
+![Edit definition](../../Assets/Screenshots/EDI/Operation/ExportDefinitionEditLineItems.png "[Edit definition]")
+
+2. Click the *File settings* tab.   
+    The *File settings* tab is displayed.
+
+     ![File settings tab](../../Assets/Screenshots/EDI/Operation/ExportDefEmptyfileSettings.png "[File settings tab]")
```

Comment:
Screenshot wird nicht angezeigt (vielleicht Groß/Kleinschreibung in Namen prüfen: file ist klein)

Reply:
Beim nächstn Projekt...